### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by Axsgranted** [kalkih/mini-media-player#782]
Title says it all - Just double checking before I switch to something else.

**Comment by Californian** [comment]
Seems like it -- any good alternatives people here would suggest?

**Comment by Boh1** [comment]
That's too bad, I really like the mini media player. I've looked and don't see anything with close to its capabilities.

**Comment by Californian** [comment]
> That's too bad, I really like the mini media player. I've looked and don't see anything with close to its capabilities.

Seems like a good time for a fork! Even half-broken, this is still better than any other card I've found (especially including the core one).

**Comment by Boh1** [comment]
> > That's too bad, I really like the mini media player. I've looked and don't see anything with close to its capabilities.
> 
> Seems like a good time for a fork! Even half-broken, this is still better than any other card I've found (especially including the core one).

Agreed. Maybe is there's a fork, at least some of the outstanding PRs that folks have graciously offered up can get resolved.
